bpe/encoder.py

```diff
@@ -23,7 +23,8 @@ class Encoder:
     """
 
     def __init__(self, vocab_size=8192, pct_bpe=0.2, word_tokenizer=None,
-                 silent=True, ngram_min=2, ngram_max=2, required_tokens=None, strict=False, 
+                 silent=True, ngram_min=2, ngram_max=2, required_tokens=None,
+                 strict=False, lowercase=False,
```

Comment:
Since the package currently lowercases, we should have `lowercase=True` to maintain compatibility for everyone already using the package.